cmd/merge/transactions.go

```diff
@@ -183,3 +211,58 @@ func mergeTransactions(cCtx *cli.Context) error {
 	log.Infof("Finished processing CSV files, wrote %s transactions", printer.Sprintf("%d", cntTxWritten))
 	return nil
 }
+
+func updateInclusionStatus(log *zap.SugaredLogger, checkNodeURIs []string, txs map[string]*common.TxSummaryEntry) (err error) {
+	// Load inclusion status for all transactions
+	workers := numRPCWorkers
+	txC := make(chan *common.TxSummaryEntry, 2000000)
+	respC := make(chan error)
```

Comment:
All workers will wait to write to this channel, I wonder if it's a bottleneck in some way. Maybe buffer here can be useful